pkg/supervisor/supervisor.go

```diff
@@ -60,6 +64,8 @@ type Supervisor struct {
 	cancel         context.CancelFunc
 }
 
+const k0sManaged = "_K0S_MANAGED=yes"
```

Comment:
does the underscore have some meaning here?

Reply:
Not really. The underscore is there because it's the value we decided during the discussion in the call.